model/src/main/proto/exploration.proto

```diff
@@ -0,0 +1,145 @@
+syntax = "proto3";
+
+import "subtitled_html.proto";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Data for a single exploration.
+// Maps from: data/src/main/java/org/oppia/data/backends/gae/model/GaeExploration.kt
+message Exploration {
+  repeated State states = 1;
+  repeated ParamChange param_changes = 2;
+  repeated ParamSpec param_specs = 3;
+  string initStateName = 4;
+  string objective = 5;
+  bool correctness_feedback_enabled = 6;
+  string title = 7;
+  string language_code = 8;
+}
+
+// Data for a param change.
+// Maps from: data/src/main/java/org/oppia/data/backends/gae/model/GaeParamChange.kt
+message ParamChange {
+  string generator_id = 1;
+  string name = 2;
+  CustomizationArgs customization_args = 3;
+}
+
+// Data for a param spec.
+// Maps from: data/src/main/java/org/oppia/data/backends/gae/model/GaeParamSpec.kt
+message ParamSpec {
+  string obj_type = 1;
+}
+
+// Data for a single state
+// Maps from: data/src/main/java/org/oppia/data/backends/gae/model/GaeState.kt
+message State {
+  // Mapping from content_id to a VoiceoverMapping
+  map<string, VoiceoverMapping> recorded_voiceovers = 1;
+  SubtitledHtml content = 2;
+  // Mapping from content_id to a TranslationMapping
+  map<string, TranslationMapping>written_translations = 3;
+  repeated ParamChange param_changes = 4;
+  string classifier_model_id = 5;
+  Interaction interaction = 6;
+  bool solicit_answer_details = 7;
```

Comment:
Similar to a comment below: this isn't as clear of a property--can you add a brief comment explaining what it is?

Reply:
Done